calendar_event_report/tests/test_weekly_report_wizard.py

```diff
@@ -0,0 +1,146 @@
+# © 2019 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
+
+from odoo.tests import common
+import datetime
+
+
+class TestWeeklyReportWizard(common.TransactionCase):
+
+    def setUp(self):
+        super(TestWeeklyReportWizard, self).setUp()
+        self.Rooms = self.env['resource.calendar.room']
+        self.WeeklyReportWizard = self.env['weekly.report.wizard']
+        self.EMPLOYEE_BASE = self.env.ref(
+            'base.group_user')
+        self.EMPLOYEE_GROUP = self.env.ref(
```

Comment:
remove this three users EMPLOYEE_GROUP, RESOURCE_CALENDAR_MANAGER and RESOURCE_CALENDAR_EDITOR and referenced tests. we can remove it in the first instance, and replace it with a test with root user